test/lib/tasks/seed_rake_test.rb

```diff
@@ -50,15 +50,15 @@ def self.glob_dummy_seeds
           ['db', 'seed', File.basename(seed_file, '.seeds.rb')].join(':')
         end.unshift('db:seed:original')
 
-        subject.prerequisites.must_equal prerequisite_seeds
+        _(subject.prerequisites).must_equal prerequisite_seeds
       end
     end
 
     describe 'when db/seeds.rb does not exist' do
       def setup
         main = TOPLEVEL_BINDING.eval('class << self; self; end')
         orig = original_seeds_file
-        main.send(:undef_method, :original_seeds_file) if main.respond_to?(:original_seeds_file, true)
+        # main.send(:undef_method, :original_seeds_file) if main.respond_to?(:original_seeds_file, true)
```

Comment:
why is this commented?

Reply:
good job 👍

Reply:
I'm curious why it started to fail

Reply:
@rkugot  tbh This is weird how we were checking this against `respond_to?` to then  try to `:undef_method` instead of something like this
```
 main.send(:undef_method, :original_seeds_file) if main.method_defined?(:original_seeds_file)
```

(I'm checking the master at the moment :) )

Reply:
Hey @m3rad I'm guessing it worked on master due to the Refined methods being unreachable with #public_send and #respond_to? up to Ruby 2.6. And `main.respond_to?(:original_seeds_file, true)` always evaluates to `false`

Reply:
@rkugot 
Yes this is the case
--> tested ruby `2.5.3` 
<img width="674" alt="Screenshot 2023-08-02 at 17 23 17" src="https://github.com/smartpension/seedbank/assets/89853750/90962be7-6455-4daf-a4ff-99d0205ebcc5">

Reply:
Looks like it makes sense to change it to the:
```ruby
main.send(:undef_method, :original_seeds_file) if main.method_defined?(:original_seeds_file)
```

Reply:
@rkugot  Thanks 
[fixed
<img width="58" alt="Screenshot 2023-08-02 at 17 56 21" src="https://github.com/smartpension/seedbank/assets/89853750/d57d04bb-b131-468c-9122-05d3cff7df1a">
](https://github.com/smartpension/seedbank/pull/2/commits/4aad041f1de0d7097cf332c3c362b55a98ef891f)